seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckConfigUtil.java

```diff
@@ -72,14 +72,14 @@ public static CheckResult checkAtLeastOneExists(Config config, String... params)
         }
     }
 
-    public static boolean isInvalidParam(Config config, String param) {
-        boolean isInvalidParam = false;
-        if (!config.hasPath(param) || config.getAnyRef(param) == null) {
-            isInvalidParam = true;
+    public static boolean isValidParam(Config config, String param) {
+        boolean isValidParam = true;
+        if (!config.hasPath(param)) {
+            isValidParam = false;
         } else if (config.getAnyRef(param) instanceof List) {
-            isInvalidParam = ((List<?>) config.getAnyRef(param)).size() == 0;
+            isValidParam = !((List<?>) config.getAnyRef(param)).isEmpty();
         }
-        return isInvalidParam;
+        return isValidParam;
```

Comment:
can be simplified to `return config.hasPath(param) && !((List<?>) config.getAnyRef(param)).isEmpty();`